src/exec/handle_io.c

```diff
@@ -84,12 +98,12 @@ static void	handle_pipeline(t_pipeline_conf *conf)
 	}
 }
 
-bool	handle_io(t_pipeline_conf *conf, t_list *redirects)
+bool	handle_io(t_pipeline_conf *conf, t_list *redirects, t_ctx *ctx)
```

Comment:
there are radical suggestions
```suggestion
bool	takeover_io(t_pipeline_conf *conf, t_list *redirects, t_ctx *ctx)
```
or
```suggestion
bool	override_io(t_pipeline_conf *conf, t_list *redirects, t_ctx *ctx)
```